metrics/api/pom.xml

```diff
@@ -88,11 +99,26 @@
                 <configuration>
                     <forceJavacCompilerUse>true</forceJavacCompilerUse>
                     <annotationProcessorPaths>
+                        <path>
+                            <groupId>io.helidon.common.features</groupId>
+                            <artifactId>helidon-common-features-processor</artifactId>
+                            <version>${helidon.version}</version>
+                        </path>
                         <path>
                             <groupId>io.helidon.config</groupId>
                             <artifactId>helidon-config-metadata-processor</artifactId>
                             <version>${helidon.version}</version>
                         </path>
+                        <path>
+                            <groupId>io.helidon.inject.configdriven</groupId>
+                            <artifactId>helidon-inject-configdriven-processor</artifactId>
```

Comment:
I am not sure we should go with configdriven here (yet)

Reply:
removed (for now)